tests/apiTests.js

```diff
@@ -0,0 +1,7 @@
+require('./queryApiTest/balance.js');
```

Comment:
Should this file be called `queryTests`? So when we add Send, Poll, Configuration... we can add a script in package.json and also add a new file for each?